package.json

```diff
@@ -31,19 +32,20 @@
   },
   "homepage": "https://github.com/openculinary/i18next-gettext-loader",
   "dependencies": {
-    "i18next-conv": "^12.1.1"
+    "glob": "11.0.3",
```

Comment:
Let's try not to introduce `glob` as an explicit dependency, since it is not imported anywhere in the library code or tests.

(and even if we do, we probably shouldn't use a fixed/pinned version number for it)